arm-apimanagement/2016-07-07/swagger/apimdeployment.json

```diff
@@ -0,0 +1,1234 @@
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiManagementServiceUploadCertificateParameters"
+            },
+            "description": "Parameters supplied to the Upload SSL certificate for an API Management service operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully uploaded certificate to the API Management Service.",
+            "schema": {
+              "$ref": "#/definitions/CertificateInformation"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }     
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/updatehostname": {
+      "post": {
+        "tags": [
+          "ApiManagementService"
+        ],
+        "operationId": "ApiManagementServices_UpdateHostname",
+        "description": "Creates, updates or deletes the custom hostnames for an API Management service. The custom hostname can be applied to Proxy and Portal endpoint. This is a long running operation and could take several minutes to complete.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ServiceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiManagementServiceUpdateHostnameParameters"
+            },
+            "description": "Parameters supplied to the UpdateHostname operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Service was successfully updated with desired hostnames.",
+            "schema": {
+              "$ref": "#/definitions/ApiManagementServiceResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "ResourcesMoveInfo": {
```

Comment:
removed it

---
In reply to: [84336584](https://github.com/Azure/azure-rest-api-specs/pull/573#discussion_r84336584) [](ancestors = 84336584)